src/Mod/AddonManager/AddonManagerOptions.ui

```diff
@@ -17,7 +17,9 @@
    <item>
     <widget class="QCheckBox" name="checkUpdates">
      <property name="toolTip">
-      <string>If this option is checked, when launching the Addon Manager, installed addons will be checked for available updates (requires the python-git package installed on your system)</string>
+      <string>If this option is selected, when launching the Addon Manager,
+installed addons will be checked for available updates
+(this requires the python-git package installed on your system)</string>
      </property>
      <property name="text">
       <string>Automatically check for updates at start (requires python-git)</string>
```

Comment:
I would use the name GitPython, which is the actual name of the project. The name `python-git` refers to a package, which may be different or non-existent in other Linux distributions.

Reply:
To be fair, per [repology](https://repology.org/project/python:gitpython/versions) python-git is fairly popular synonym for GitPython and so this is somewhat trivial, but nevertheless aeaa3a1